modules/Blockchain/Polygon/contracts/XDAITestTrac.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity 0.4.25;
+
```

Comment:
Can we remove this contract from Polygon implementation since it's connected to xDAI?